app/view_models/profile_index.rb

```diff
@@ -16,13 +16,21 @@ def personal_key_partial
   end
 
   def password_reset_partial
-    if current_user.password_reset_profile.present?
+    if current_user.decorate.password_reset_profile.present?
```

Comment:
`DecoratedUser` does not delegate to `User`.

So currently it's a choice between:
```ruby
decorated_user.user.foo
decorated_user.password_reset_profile.present?
# or
current_user.foo
current_user.decorate.password_reset_profile.present?
```
which seems about 6 of one, half dozen of the other to me.